lamkey_nick_ndvi.py

```diff
@@ -0,0 +1,723 @@
+#!/usr/bin/env python
```

Comment:
@nlamkey codes run on all os per github ci checks (assume this also means the code runs from start to finish as well, checking off another box in the review checklist).